release-1.8/release_notes_draft.md

```diff
@@ -44,3 +44,37 @@ please check out the [release notes guidance][] issue.
 ## **Deprecations**
 
 ## **Notable Features**
+
+### **Node Components**
+#### kubelet
```

Comment:
@timothysc oh I see, sorry. Just found out about this hidden process since the release-note tags in feature PRs are apparently not used for anything now.